src/app/login/login.component.css

```diff
@@ -1,94 +1,96 @@
-*{box-sizing:border-box;}
```

Comment:
Global styles are set in the top-level css file (styles.css). Box-sizing is set for all elements in that file.